examples/display_data.py

```diff
@@ -12,15 +12,39 @@
 """
 
 from parlai.core.params import ParlaiParser
+from parlai.core.agents import Agent
 from parlai.agents.repeat_label.repeat_label import RepeatLabelAgent
 from parlai.core.worlds import create_task
 
 import random
 
+
+class RepeatTextAgent(Agent):
+    """Simple agent which repeats the text it receives."""
+    def __init__(self, opt, shared=None):
+        super().__init__(opt)
+        self.id = 'RepeatTextAgent'
+
+    def act(self):
+        obs = self.observation
+        if obs is None or 'text' not in obs:
+            return {'text': 'Nothing to repeat yet.'}
+        return {'id': self.getID(), 'text': obs['text']}
+
+
 def display_data(opt):
-    # create repeat label agent and assign it to the specified task
-    agent = RepeatLabelAgent(opt)
-    world = create_task(opt, agent)
+    # create one repeat label agent and assign it to the specified task
+    agents = []
+    agents.append(RepeatLabelAgent(opt))
+
+    # since RepeatLabelAgent action has label in 'text', other agents
```

Comment:
i don't really understand this, what text are they repeating exactly? the same that the first repeat label agent said? not sure i understand the use case exactly..

Reply:
@jaseweston yes you are right, they repeat what the `RepeatLabelAgent` said. Let me clarify it in a separate comment.